### DISCUSSION TITLE
请问如何修改默认播放控件功能

### DISCUSSION THREAD

**Discussion by iammsf** [bytedance/xgplayer#820] [Q&A]
我在我的Vue项目中导入了xgplayer-vue，项目运行完美。
但是我想要自定义播放和暂停等按键的功能，直接在xgplayer/dist/control/play.js和xgplayer/src/control/play.js中修改源代码，在Player.pause（）后加入console.log（）进行测试，发现播放暂停后并不会触发log事件。。。
es和browser文件夹我也尝试修改了，都无法达到目的。请问到底该作何修改才能正常触发

**Comment by yiwen03** [comment]
可以修改dist index.js文件，但是不建议修改node_modules下的代码，如果需要，可以自己定义play pause插件，然后集成到播放器里，https://v2.h5player.bytedance.com/plugins/#%E8%87%AA%E5%AE%9A%E4%B9%89%E6%8F%92%E4%BB%B6

**Comment by iammsf** [comment]
> 可以修改dist index.js文件，但是不建议修改node_modules下的代码，如果需要，可以自己定义play pause插件，然后集成到播放器里，https://v2.h5player.bytedance.com/plugins/#%E8%87%AA%E5%AE%9A%E4%B9%89%E6%8F%92%E4%BB%B6

问题就在于官方文档太简略了啊，啥啥都没有。
只说定义一个js文件，但是并没有说明要放在哪个目录里，如何引入。直接在初始化代码里配置插件，播放器初始化完成之后确实可以执行插件逻辑，但是怎么才能让插件在control bar里显示呢？文档真的一言难尽
